pkgs/development/lisp-modules-new/imported.nix

```diff
@@ -35997,6 +35997,7 @@ in {
     });
     systems = [ "gsll" ];
     lispLibs = [ (getAttr "alexandria" pkgs) (getAttr "foreign-array" pkgs) (getAttr "cffi-grovel" pkgs) (getAttr "cffi-libffi" pkgs) (getAttr "lisp-unit" pkgs) (getAttr "metabang-bind" pkgs) (getAttr "trivial-features" pkgs) (getAttr "trivial-garbage" pkgs) ];
+    nativeLibs = [ nativePkgs.gsl nativePkgs.pkg-config ];
```

Comment:
so not sure how to really fix it
```patch
diff --git a/pkgs/development/lisp-modules-new/lisp-packages.nix b/pkgs/development/lisp-modules-new/lisp-packages.nix
index 513f72a985e..cf909400a05 100644
--- a/pkgs/development/lisp-modules-new/lisp-packages.nix
+++ b/pkgs/development/lisp-modules-new/lisp-packages.nix
@@ -206,6 +206,8 @@ let
         # Similiarily for native deps
         export LD_LIBRARY_PATH=${makeLibraryPath nativeLibs}:$LD_LIBRARY_PATH
         export CLASSPATH=${makeSearchPath "share/java/*" javaLibs}:$CLASSPATH
+        export CFLAGS="${lib.concatStringsSep " " (map (lib: "-I ${lib}/include") nativeLibs)}"
+        echo "$CFLAGS"
 
         # Make asdf compile from `src` to pwd and load `lispLibs`
         # from storeDir. Otherwise it could try to recompile lisp deps.

```
with this it still doesn't try to add `-I`
```
building
-I /nix/store/rf4lq6rywjjyj70msallqrnrmhf0anjq-gsl-2.7.1/include
; compiling file "/nix/store/zyac8y9jc298f84rhwxr2xp6q0vvadkj-source/init/init.lisp" (written 01 JAN 1970 12:00:01 AM):

; wrote /build/source/init/init-tmpGHU3ALSV.fasl
; compilation finished in 0:00:00.013
; gcc -o /build/source/init/libgsl-unix__grovel-tmpAAURSO1.o -c -g -Wall -Wundef -Wsign-compare -Wpointer-arith -O3 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -Wunused-parameter -fno-omit-frame-pointer -momit-leaf-frame-pointer -fPIC -I/nix/store/1q2rmnnzpq9drpzj0g2gidrnpk1jqlzs-cffi-grovel-0.24.1/ /build/source/init/libgsl-unix__grovel.c
/build/source/init/libgsl-unix__grovel.c:7:10: fatal error: gsl/gsl_integration.h: No such file or directory
    7 | #include <gsl/gsl_integration.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

Reply:
marking it back as broken but with updated comment. Not sure how to make it build atm